hles/extraction/src/it/scala/org/broadinstitute/monster/dap/ExtractionPipelineBuilderIntegrationSpec.scala

```diff
@@ -57,7 +57,7 @@ class HLESurveyExtractionPipelineBuilderIntegrationSpec extends PipelineBuilderS
 
   it should "only download records that have completed all HLES instruments" in {
     readMsgs(outputDir, "records/*.json").foreach { record =>
-      HLESurveyExtractionPipelineBuilder.ExtractionFilters
+      HLESurveyExtractionPipeline.HLESExtractionFilters
```

Comment:
I think the CI is getting tripped up by this line
`ExtractionPipelineBuilderIntegrationSpec.scala:60:35: value HLESExtractionFilters is not a member of object org.broadinstitute.monster.dap.HLESurveyExtractionPipeline`

Reply:
Fix is on the way, good catch!